students/daniel_carrasco/session02/grid_print.py

```diff
@@ -0,0 +1,16 @@
+def print_grid(n,x):
+    space = x*' - '
+    l = ('+'+x*' - ')*n+'+'
```

Comment:
use actual names and not chars for variable names: `length`, `width`, and I don't know what `j` stand for :)